test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -503,6 +503,50 @@ Describe "Get-Help should accept arrays as the -Parameter parameter value" -Tags
     }
 }
 
+Describe "Get-Help for function parameter should be consistent" -Tags 'CI' {
+    BeforeAll {
+        $test1 = @'
+            function test1 {
+                param (
+                    $First
+                )
+            }
+'@
+        $test2 = @'
+            function test2 {
+                param (
+                    $First,
+                    $Second
+                )
+            }
+'@
+        $test1Path = Join-Path $TestDrive "test1.ps1"
+        New-Item -ItemType File -Path $test1Path -Force > $null
```

Comment:
Don't need this. `Set-Content` will create a file.

Reply:
Done. :)